docs/telemetry.md

```diff
@@ -83,7 +83,7 @@ The event ping contains a list of events ([see event format on readthedocs.io](h
 (*)When the pin site switch is clicked, the state (on/off) of the desktop mode switch is also sent.
 
 (**)YouTube tile clicks are a sub-category of bundled tile clicks- each click will count towards both values.
-This value existed from v3.2 to v3.8. For versions after v3.8, the tile_id is included as an extra, not as a value.
+This value existed from v3.2+, but is intended for removal. For versions after v3.8, the tile_id is included as an extra.
```

Comment:
nit: -> "For versions after v3.8, the tile_id is included as an extra but, as of v3.10, this new probe does not accurately match the original YouTube probe: see #2091 for a discussion"

It's not necessary to add but it could be helpful to make it clear that that may not be the final implementation. I wonder if that means counting clicks on our new channel tiles may be inaccurate... I'll follow up with Ashley.

Reply:
Actually, if I find a bug in the implementation, it's probably better for me to update that as part of #2091 so nevermind about updating it here.